base/stacktraces.jl

```diff
@@ -268,22 +268,29 @@ function show(io::IO, frame::StackFrame; full_path::Bool=false)
     end
 end
 
+function Base.parentmodule(frame::StackFrame)
+    if frame.linfo isa MethodInstance
+        def = frame.linfo.def
+        if def isa Module
+            return def
+        else
+            return (def::Method).module
+            return def.module
+        end
+    else
+        # Bug: currently the module is not available for inlined frames and
```

Comment:
```suggestion
        # the module is not always available (common reasons include inlined frames and frames arising from the interpreter)
```
Not a bug for debug information to be incomplete

Reply:
Fair enough, will change. It's not a bug itself (though can easily lead to bugs).